### ISSUE TITLE
Seminar form: hybrid or online

### ISSUE THREAD

**Issue by xp1632** [xp1632/DFKI_working_log#32]
- Due to some administration arrangements, I need to have plans for a hybrid or online or offline setting for the presentation

---
- After some thoughts:

- I won't prepare for a [hybrid] presentation, because it's tedious to prepare and not best performance for the online audience

---

-【If Philippe online】， I'll do it online.
- the advantages are 
   - I can check the notes in presenter mode easily 

- 【If Philippe offline】, I'll do it offline.
- the advantages are 
    - I can show myself better as a whole person

---

- In conclusion, I'll prepare a better mic as copilot suggests:

- I prefer a lapel mic
- This better mic I can also use it to record my voice for video demo

**Comment by xp1632** [comment]
### Share the sound as well in Zoom 

---
Zoom:

https://cs-uni-saarland-de.zoom.us/j/82820719300?pwd=aaFg0aoFBEMongUprIE6QAE8VuAyRh.1

 

Meeting-ID: 828 2071 9300

Kenncode: 080424
---

- I was given a Zoom meeting, with Zooms, I need to be careful about the setting of sharing screen to ensure that the sound of my live demo video would be shared as well.

---
![image](https://github.com/xp1632/DFKI_working_log/assets/8528052/f34e8c63-ea1d-421d-86cc-2e08908ea4ed)

https://mitsloanedtech.mit.edu/support/how-to-share-and-play-a-video-with-sound-during-a-zoom-meeting/

**Comment by xp1632** [comment]
After testing, Zoom is also okay, and I can even add some beautifying filters for my face, haha
